Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -459,7 +459,7 @@ script:
       description: List of collections.
       type: String
   - arguments:
-    - default: true
+    - default: false
```

Comment:
why are we setting that one to `false` as well? what is the behavior currently (before this change)? what happens if we pass a default arg?

Reply:
this one is by mistake.